### ISSUE TITLE
link in menus header missing `aria-label `

### ISSUE THREAD

**Issue by Weaxs** [nunocoracao/blowfish#1904]
**Describe the bug**
[Links must have discernible text](https://dequeuniversity.com/rules/axe/4.10/link-name)

menue.toml:
```
[[main]]
identifier = "travelling"
pre = "travelling"
url = "https://www.travellings.cn/go.html"
weight = 60

```

html in website:

```
<a href="https://www.travellings.cn/go.html" target="_blank" class="flex items-center text-gray-500 hover:text-primary-600 dark:hover:text-primary-400">
    
    <span>
        

  <span class="relative block icon">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512">
    <path fill="currentColor" d="M96 0C43 0 0 43 0 96V352c0 48 35.2 87.7 81.1 94.9l-46 46C28.1 499.9 33.1 512 43 512H82.7c8.5 0 16.6-3.4 22.6-9.4L160 448H288l54.6 54.6c6 6 14.1 9.4 22.6 9.4H405c10 0 15-12.1 7.9-19.1l-46-46c46-7.1 81.1-46.9 81.1-94.9V96c0-53-43-96-96-96H96zM64 128c0-17.7 14.3-32 32-32h80c17.7 0 32 14.3 32 32v96c0 17.7-14.3 32-32 32H96c-17.7 0-32-14.3-32-32V128zM272 96h80c17.7 0 32 14.3 32 32v96c0 17.7-14.3 32-32 32H272c-17.7 0-32-14.3-32-32V128c0-17.7 14.3-32 32-32zM64 352a32 32 0 1 1 64 0 32 32 0 1 1 -64 0zm288-32a32 32 0 1 1 0 64 32 32 0 1 1 0-64z"></path>
</svg>
  </span>


    </span>
    
    <p class="text-base font-medium" title="">
        
    </p>
</a>
```

`<a>` missing `aria-label ` attribute

**Screenshots**
![image](https://github.com/user-attachments/assets/13a0fcbd-9e3f-46a4-b060-a27a740e2a46)

**Hugo & Blowfish versions**

blowfish version` v2.78.0`

**Comment by ZhenShuo2021** [comment]
Fixed in 6c8e0d7933a81a5d2b3183137c327872644297a8
